grails-app/controllers/org/pih/warehouse/api/ProductSupplierApiController.groovy

```diff
@@ -28,4 +30,17 @@ class ProductSupplierApiController {
         productSupplierService.delete(params.id)
         render status: 204
     }
+
+    def create(ProductSupplierDetailsCommand productSupplierCommand) {
+        ProductSupplier productSupplier = productSupplierService.saveDetails(productSupplierCommand)
+
+        response.status = HttpStatus.CREATED.value()
```

Comment:
This looks hardcore, I think it is safe to use numbers for HTTP status codes :sweat_smile:

Reply:
I think I will retract my comment....

The reason I said this is that I usually dealt with HTTP codes in a number format and they are the same across all frameworks, and importing the HttpsStatus and extracting that exact number using `HttpStatus.CREATED.value()` was something that caught my attention, but thinking about this right now I think I see a point in this, this is probably easier to read and debug 

soooo... statement rollback, this looks right :+1: